targeting_terminal.py

```diff
@@ -0,0 +1,52 @@
+import pygame
+
+pygame.font.init()
+
+# CONSTANTS
+BLACK = 0, 0, 0
+BLUE = 0, 0, 255
+WHITE = 255, 255, 255
+EVENT_KEY_a = 97
+EVENT_KEY_z = 122
+EVENT_KEY_ENTER = 13
+EVENT_KEY_BACKSPACE = 8
+fontSize = 15
+rectX = 10
+rectY = 40
+rectWidth = 400
```

Comment:
Instead of using `rect` I'd suggest using `terminal`, `rect` could be the rect for anything.